ten-simple-rules-dockerfiles.Rmd

```diff
@@ -95,7 +96,7 @@ Furthermore, workflows interacting with many petabytes of data and executed in h
 Using such HPC job managers or cloud infrastructures would require a collection of "Ten Simple Rules" articles in their own right. 
 For the HPC use case, we encourage the reader to look at Singularity [@kurtzer_singularity_2017].
 For this article, we focus on workflows that typically run on single machine, e.g., a researchers laptop computer or a virtual server.
-The reader might scope the data requirement to less than a terabyte, and compute requirement to a machine with 16 cores running over the weekend.
+The reader might scope the data requirement to under a terabyte, and compute requirement to a machine with 16 cores running over the weekend.
```

Comment:
Question: do you think a terabyte is really a reasonable size that someone could easily work with, locally? My personal computer has never had more than 500GB. The only way I could work with that size of data would be to move to our local cluster, and I think earlier we already mentioned this paper is out of scope to talk about HPC best practices (e.g., because the nodes get purged at some regularity, $HOME only has 15GB and is bad to cache containers there, Docker of course doesn't work, that sort of thing)

Reply:
I understand where you're coming from. I would nevetheless like to stick to this loose wording. If you're analysing remote sensing data, for example, you will have a desktop computer with a storage of 4-8 TB to do handle such size, especially if you're not a developer. For archeologists or geographers, buying another disk and waiting is often a better choice than learning how to work with a (local) cluster.

@sje30 I know you just made a small wording change, but maybe you can share your experiences here?

Reply:
I think an absolute number is always to going to be difficult given different laptop/workstations and data sizes that people work with.  But I think I would stick with under a TB as a rough guide ( just using orders of magnitude, I guess we could say  "tens/hundreds of Gbs", but that looks a bit clumsy.)  I suggest we not worry about it.  Given we have existence proofs of people work with sub TB data, I think we are fine.

Reply:
That's reasonable, work for me. Maybe someday I will have such a large workstation! :)